.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 100,
+  "trailingComma": "all",
+  "arrowParens": "avoid"
```

Comment:
I would say that this is a highly subjective point of view. For instance I do find it clearer without parenthesis, and I find the reasons in paragraph that you quoted dubious.

>  it becomes harder to add type annotations, extra arguments or default values as well as making other changes

Is it that hard to add a parenthesis manually when needed? And remove the visual noise otherwise automatically?

But in the end this kind of nitpicking over a tiny formatting style is completely unimportant to me. I would glady take whatever setting the whole team find best (even though I am sure most people actually do not care at all).